File: Is Subsequence/code.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        int l1=0,l2=0;
        while(l1<s.size() && l2<t.size())
        {
            if(s[l1] == t[l2])
            {
                l1++;
                l2++;
            }
            else
            l2++;
        }
        if(l1 == s.size())
        return true;
        else
        return false;
    }
};

File: Remove Duplicates/code.java
class Solution {
    String removeDups(String S) {
        // code here
          String S1="";
        ArrayList<Character> al=new ArrayList<>();
        for(int i=0;i<S.length();i++)
        {
            if(!(al.contains(S.charAt(i))))
            al.add(S.charAt(i));
        }
        for(int i=0;i<al.size();i++)
        S1=S1+al.get(i);
        return(S1);
    }
}
